ext4/tar2ext4/tar2ext4.go

```diff
@@ -312,3 +272,15 @@ func ConvertAndComputeRootDigest(r io.Reader) (string, error) {
 	hash := dmverity.RootHash(tree)
 	return fmt.Sprintf("%x", hash), nil
 }
+
+// ConvertToVhd converts given io.ReadWriteSeeker to VHD, by appending the VHD footer with a fixed size.
+func ConvertToVhd(w io.ReadWriteSeeker) error {
+	size, err := w.Seek(0, io.SeekEnd)
+	if err != nil {
+		return err
+	}
+	if err = binary.Write(w, binary.BigEndian, makeFixedVHDFooter(size)); err != nil {
+		return nil
```

Comment:
```suggestion
		return err
```

or just replace 282-285 with the below.
```go
return binary.Write(w, binary.BigEndian, makeFixedVHDFooter(size))
```